.github/pull_request_template.md

```diff
@@ -25,6 +25,7 @@ Please answer these questions before creating your pull request. Thanks!
    - [ ] I am adding a new dependency
    - [ ] If this is a new feature/behavior, I'm adding the Local Testing parity changes.
    - [ ] I acknowledge that I have ensured my changes to be thread-safe. Follow the link for more information: [Thread-safe Developer Guidelines](https://github.com/snowflakedb/snowpark-python/blob/main/CONTRIBUTING.md#thread-safe-development)
```

Comment:
let's not link to an internal document here, it's an open source repo :)

Can we maybe add a doc on what to do re AST and add it to the repo? This should have a workflow for both internal and external contributors.